src/azure-cli/azure/cli/command_modules/netappfiles/custom.py

```diff
@@ -426,14 +421,9 @@ def pre_operations(self):
 class VolumeUpdate(_VolumeUpdate):
     @classmethod
     def _build_arguments_schema(cls, *args, **kwargs):
-        from azure.cli.core.aaz import AAZStrArg, AAZIntArgFormat
+        from azure.cli.core.aaz import AAZIntArgFormat
         args_schema = super()._build_arguments_schema(*args, **kwargs)
-        args_schema.vnet = AAZStrArg(
-            options=["--vnet"],
-            arg_group="Properties",
-            help="Name or Resource ID of the vnet. If you want to use a vnet in other resource group or subscription, please provide the Resource ID instead of the name of the vnet.",
-            required=False,
-        )
```

Comment:
It's a breaking change to remove the vnet argument in update command.

Reply:
true, this parameter is not needed/used in update.  We can add it back to avoid breaking as that change is not related to the intended fix.